### PR TITLE
Topic/av test2

### PR DESCRIPTION
**Pull request by hppritcha**
add new av_test to unit tests dir.  this test should pass for gni provider.

@sungeunchoi

### CODE REVIEW

**Comment by bturrubiates** [comment]
:+1: @sungeunchoi If this looks good to you, I'll merge it.

**Comment by sungeunchoi** [comment]
This is fine, but the new file should have LANL copyright, no?

File: unit/av_test2.c [hunk 1]
```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright (c) 2013-2014 Intel Corporation.  All rights reserved.
+ * Copyright (c) 2014 Cisco Systems, Inc.  All rights reserved.
+ *
```
Comment (bturrubiates) [inline unit/av_test2.c:4]: This should probably be cray/LANL

**Comment by hppritcha** [comment]
oh yes I'll drop the intel/cisco and put in cray.

**Pull request merged**